docs/components/Balance.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 2
+---
+
+# Balance
+
+Displays the balance of a given address in both ether (ETH) and US dollars (USD).
+
+![Balance Example](/img/Balance.png)
```

Comment:
Maybe we should put a pic without any styling? Like just how Balance component would look if they use : 

```tsx
<Balance address="0x34aA3F359A9D614239015126635CE7732c18fDF3" />
```

and maybe use GIF 😅 (really sorry for nitpicking hard) even current one looks good and we can ignore it 


![balanceComponent](https://github.com/scaffold-eth/se2-docs/assets/80153681/fb47d0fd-fdc3-4ed7-8656-2a0d1dbc5f1c)